.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,34 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          container-runtime: remote
+          runtime-cgroups: /system.slice/crio.service
       ---
-      kind: KubeletConfiguration
       apiVersion: kubelet.config.k8s.io/v1beta1
-      cgroupDriver: cgroupfs
+      kind: KubeletConfiguration
+      cgroupDriver: systemd
+      cgroupRoot: /
+      systemCgroups: /system.slice
+      kubeletCgroups: /system.slice/kubelet.service
+      enforceNodeAllocatable:
+      - pods
+      #- kube-reserved # try again when we bump system to get cgroupsv2
```

Comment:
Do you think you could remove the commented out lines and create a tracking issue instead? With a note saying why you couldn't do it here?

Reply:
I'll try to restructure it into multiple PRs, hopefully the next one being able to be based on top of ubuntu 22.04 bump and see where we stand